doc/source/add-node.rst

```diff
@@ -15,17 +15,14 @@ it automatically joins the cluster and synchronizes with it.
    Do not join several nodes at the same time
    to avoid overhead due to large amounts of traffic when a new node joins.
 
-By default, |PXC| uses |PXB|_ for *State Snapshot Transfer* (:term:`SST`).
+By default, |PXC| uses |PXB|_ for :term:`State Snapshot Transfer <SST>`.
 This requires the following:
 
-* Set the :variable:`wsrep_sst_method`` variable to ``xtrabackup-v2`` and
-  provide SST user credentials with the :variable:`wsrep_sst_auth`` variable.
-
+* Set the :variable:`wsrep_sst_method`` variable to ``xtrabackup-v2``.
   For more information, see :ref:`configure`.
+* Create a user for SST on the initial node. For more information, see
```

Comment:
We do not need to "Create a user for SST on the initial node."  This was the user in the wsrep_sst_auth.